solidity/contracts/NetworkSettings.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "./INetworkSettings.sol";
+import "./utility/Owned.sol";
+import "./utility/Utils.sol";
+
+/**
+ * @dev This contract maintains the network settings.
+ */
+contract NetworkSettings is INetworkSettings, Owned, Utils {
+    uint32 private constant PPM_RESOLUTION = 1000000;
+
+    address private _feeWallet;
```

Comment:
even though these aren't called from a contract, we should still use names that provide the context.
we might introduce other types of fees like affiliate fee here.
I'd also prefer to use just fee and not "portion", I think it's easier to understand and in line with the (conversion) fee in the pools.

Reply:
So what name do you wanna use?

Reply:
networkFee
networkFeeWallet

Reply:
Done.